File: DNN Platform/Admin Modules/Dnn.Modules.ModuleCreator/Templates/C#/Web Service API/templateRouteMapper.cs
// 
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the MIT License. See LICENSE file in the project root for full license information.
// 
#region Using Statements

using DotNetNuke.Web.Api;

#endregion

namespace _OWNER_._MODULE_
{
    public class _MODULE_RouteMapper : IServiceRouteMapper
    {
        public void RegisterRoutes(IMapRoute mapRouteManager)
        {
            mapRouteManager.MapHttpRoute("_OWNER_._MODULE_", "default", "{controller}/{action}", new[] {"_OWNER_._MODULE_"});
        }
    }
} 

File: DNN Platform/Admin Modules/Dnn.Modules.ModuleCreator/Templates/VB/Web Service API/templateRouteMapper.vb
#Region "Copyright"

' 
' Copyright (c) _YEAR_
' by _OWNER_
' 

#End Region

#Region "Using Statements"

Imports DotNetNuke.Web.Api

#End Region

Namespace _OWNER_._MODULE_
    Public Class _MODULE_RouteMapper
        Implements IServiceRouteMapper
        Public Sub RegisterRoutes(mapRouteManager As IMapRoute)
            mapRouteManager.MapHttpRoute("_OWNER_._MODULE_", "default", "{controller}/{action}", New () {"_OWNER_._MODULE_"})
        End Sub
    End Class
End Namespace
